src/components/Grid/Grid.scss

```diff
@@ -0,0 +1,3 @@
+.Grid {
+  height: 100vh !important;
```

Comment:
This height will need to change down the line as the Menu Bar will take up some of that height. We might just need to set `.root` to 100vh, and have the canvas fill up the remainder of the space